doc/modules/model_evaluation.rst

```diff
@@ -707,17 +707,21 @@ The :func:`hamming_loss` computes the average Hamming loss or `Hamming
 distance <https://en.wikipedia.org/wiki/Hamming_distance>`_ between two sets
 of samples.
 
-If :math:`\hat{y}_j` is the predicted value for the :math:`j`-th label of
-a given sample, :math:`y_j` is the corresponding true value, and
-:math:`n_\text{labels}` is the number of classes or labels, then the
-Hamming loss :math:`L_{Hamming}` between two samples is defined as:
+If :math:`\hat{y}_{i,j}` is the predicted value for the :math:`j`-th label of a
+given sample point :math:`i`, :math:`y_{i,j}` is the corresponding true value,
+:math:`n_\text{samples}` is the number of sample points and
+:math:`n_\text{labels}` is the number of labels, then the Hamming loss 
+:math:`L_{Hamming}` is defined as:
```

Comment:
I would rather use "sample" instead of "sample points", but otherwise this PR LGTM.